src/components/Treebank/Treebank.module.css

```diff
@@ -1,3 +1,14 @@
 .treebankContainer {
   position: relative;
 }
+
+.text p {
+  margin-bottom: 0.7rem;
+  margin-top: 0.7rem;
+}
+
+.graph {
+  display: flex;
+  height: 60vh;
+  min-height: 500px;
```

Comment:
I picked styles that looked good to me on windows of different sizes. In a later release, when we entirely replace `arethusa-widget`, we can make the whole page `flex` and revisit the styling.